### PR TITLE
Added Xue Bin Peng to SFU

### PR DESCRIPTION
**Pull request by xbpeng**
Added Xue Bin Peng to SFU

### CODE REVIEW

File: csrankings-x.csv [hunk 1]
```diff
@@ -371,6 +371,7 @@ Xue Zhang 0001,UESTC,https://www.is.uestc.edu.cn/teachers.do?id=1079,NOSCHOLARPA
 Xue-Song Qiu 0001,BUPT,https://scs.bupt.edu.cn/info/1295/2728.htm,NOSCHOLARPAGE
 Xue-Zhong He,University of Technology Sydney,https://www.uts.edu.au/staff/tony.he1,x2Q5BscAAAAJ
 Xue-song Qiu 0001,BUPT,https://scs.bupt.edu.cn/info/1295/2728.htm,NOSCHOLARPAGE
+Xue Bin Peng, Simon Fraser University,https://xbpeng.github.io/,FwxfQosAAAAJ&hl
```
Comment (emeryberger) [inline csrankings-x.csv]: Remove spaces after commas. Also, please confirm >= 75% appointment at SFU.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Review by xbpeng — COMMENTED** [review]
removing comma. I can confirm that I have a full time position at SFU (https://www.sfu.ca/computing/people/faculty/jason-peng.html).

**Comment by emeryberger** [comment]
Since you didn't say "75%+", I have to ask again: please confirm this is the case. Thanks.

**Comment by xbpeng** [comment]
I confirm my position at SFU is 75%+.

**Pull request merged**